### DISCUSSION TITLE
Importing css from nuxt config not allowed

### DISCUSSION THREAD

**Discussion by danvega** [nuxt/nuxt#17561] [Questions]
In nuxt.config.ts I am importing a css file so that is available everywhere like so: 

```
export default defineNuxtConfig({
  build: {
    postcss: {
      postcssOptions: {
        plugins: {
          tailwindcss: {},
          autoprefixer: {},
        },
      },
    },
  },
  css: [
    "~/assets/css/tailwind.css"
  ],
})
```

When I do this everything works but I am seeing this error in the console. Is there a better way to import this CSS file so Tailwind is available everywhere? 

 ERROR  Importing directly from a nuxt.config file is not allowed. Instead, use runtime config or a module. [importing /nuxt.config.ts from index.html]

**Comment by TheAlexLichter** [comment]
Please share a CodeSandbox or StackBlitz. This would make pinpointing the issue easier 😋 

CSS import in the `nuxt.config.ts` seem to be fine. But you likely import the `nuxt.config.ts` somewhere in your project (which is not allowed). Could that be?

See https://github.com/nuxt/framework/issues/1471 for more info regarding the error message you received
